test/test_helper.bash

```diff
@@ -3,7 +3,7 @@ export TMP="$BATS_TEST_DIRNAME/tmp"
 if [ "$FIXTURE_ROOT" != "$BATS_TEST_DIRNAME/fixtures" ]; then
   export FIXTURE_ROOT="$BATS_TEST_DIRNAME/fixtures"
   export INSTALL_ROOT="$TMP/install"
-  PATH=/usr/bin:/usr/sbin:/bin/:/sbin
+  PATH="/usr/bin:/usr/sbin:/bin/:/sbin:$PATH"
```

Comment:
Which important directory was missing from this list? I specifically didn't want to include the whole `$PATH`.